internal/controlplane/handlers_githubwebhooks_test.go

```diff
@@ -3331,7 +3331,9 @@ func (s *UnitTestSuite) TestHandleGitHubAppWebHook() {
 				require.Equal(t, providerID, evt.ProviderID)
 				require.Equal(t, projectID, evt.ProjectID)
 				require.Equal(t, v1.Entity_ENTITY_REPOSITORIES, evt.EntityType)
-				require.Equal(t, "stacklok/minder", evt.Properties[properties.PropertyName])
+
+				// the name can be either stacklok/minder or stacklok/trusty
+				require.Contains(t, []string{"stacklok/minder", "stacklok/trusty"}, evt.Properties[properties.PropertyName])
```

Comment:
**question:** I think the change is fine, we don't care about the order here, but I am curious, do you know why this happens?

We use go channels in these tests as backend for Watermill, which should guarantee order, so there must be something that randomly switches order in the caller, I guess. Maybe some data structure?

Reply:
I'm not sure yet. There are two options:
* we're getting packages in an unordered fashion for some reason (some map iteration maybe? these are unordered in golang).
* we're losing a message.

Either way, we should probably add an assertion about the amount of messages processed if we haven't yet.